nginx.conf

```diff
@@ -18,6 +18,9 @@ http {
     # Proxy Caching (Optional)
     include /etc/nginx/conf/nginx_cache_http.conf;
 
+    # Big Buffers (Optional)
+    include /etc/nginx/conf/upload_size*.conf;
```

Comment:
should this be `/etc/nginx/conf/nginx_big_buffers.conf`?

Reply:
Yeah. That might explain a few things. 🤦